### PR TITLE
Spotify 50 new releases - Arianna & Åsa

### PR DESCRIPTION
**Pull request by ariallahyar**
Pair-programming project with Åsa.
Link to our deployed project: https://spotifynewreleases2022.netlify.app/

### CODE REVIEW

File: code/src/components/Artist.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+
+export const Artist = (props) => {
+
+	if (props.name.length === 1) {
+		return (
+			<div>
+				<a href={props.url[0]} alt=''>
+					<h3 className="artistsName artistName">{props.name[0]}</h3>
+				</a>
```
Comment (savannah-hayes) [inline code/src/components/Artist.js]: `props.url[0]` and `props.name[0]` seems to be used more than once. Have you thought about creating constants for both of these?

File: code/src/components/Artist.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+		const lastArtist = props.name.pop();
+		const lastLink = props.url.pop();
+
+		return (
+			<h3 className='artistsName'>
+
+				{props.name.map((name) => {
+					return <><a className='artistName' href={props.url[props.name.indexOf(name)]}>{name}</a>, &nbsp;</>
+				})
```
Comment (savannah-hayes) [inline code/src/components/Artist.js]: Clever use of these Entities Codes. I assume &nbsp means to add a space. That was smart!

File: code/src/components/Artist.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+					<a className='artistName' href={props.url[1]} alt=''>{props.name[1]}</a>
+				</h3>
+			</div>
+		)
+
+	} else {
+
+		const lastArtist = props.name.pop();
+		const lastLink = props.url.pop();
+
```
Comment (savannah-hayes) [inline code/src/components/Artist.js]: Good job declaring constants here, it made it easier to follow your code and I can see clearly where you are using them.

File: code/src/data-new.json [hunk 4]
```diff
@@ -0,0 +1,11318 @@
...
+          {
+            "external_urls": {
+              "spotify": "https://open.spotify.com/artist/6jPriOYMyUL9kNonBI1CNd"
+            },
+            "href": "https://api.spotify.com/v1/artists/6jPriOYMyUL9kNonBI1CNd",
+            "id": "6jPriOYMyUL9kNonBI1CNd",
+            "name": "Faustix",
+            "type": "artist",
+            "uri": "spotify:artist:6jPriOYMyUL9kNonBI1CNd"
+          },
```
Comment (savannah-hayes) [inline code/src/data-new.json:17]: Really nice adding the latest music releases in a new json file!

File: code/src/index.css [hunk 5]
```diff
@@ -1,13 +1,118 @@
...
+  .albumsWrapper {
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+@media screen and (min-width: 1024px) {
+  .albumsWrapper {
+    grid-template-columns: repeat(4, 1fr);
+  }
+}
```
Comment (savannah-hayes) [inline code/src/index.css:118]: Your css code is very nicely structured and I have no comments other than that it looks great!

**Review by savannah-hayes — APPROVED** [review]
Good job Åsa and Arianna on your project. Your code is easy to understand, clean, and there are no bugs. I would just say that it could be a bit more dry, meaning there could be less code. The structure of your code was nice to follow and easy to read. 

You project definitely meets all the requirements and you guys were able to accomplish extra. I really enjoyed seeing how you solved the (, &) between the artists names with JavaScript. Great job overall!

File: code/src/components/Artist.js [hunk 6]
```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+export const Artists = ({ artists }) => {
+
+  if (artists.length === 1) {
```
Comment (rawisou) [inline code/src/components/Artist.js:5]: Nice way to solve the & , challenge.

**Pull request closed without merging**